desktop/ui/src/main/java/org/datacleaner/widgets/properties/AbstractMultipleCheckboxesPropertyWidget.java

```diff
@@ -173,6 +173,27 @@ protected JCheckBox addCheckBox(E item, boolean checked) {
         return checkBox;
     }
 
+    protected void editCheckBox(E oldvalue, E newValue) {
+        final String name = getName(oldvalue);
+        final DCCheckBox<E> checkBox = _checkBoxes.get(name);
+        if (checkBox != null) {
+            _checkBoxes.remove(name);
+            checkBox.addListener(CHANGE_LISTENER);
+            final boolean isSelected = checkBox.isSelected();
+            checkBox.setValue(newValue);
+            checkBox.setSelected(isSelected, true);
+            _checkBoxes.put(name, checkBox);
+            add(checkBox);
+            /* 'fireValueChanged' changes the value in the component builder. Otherwise the listener is
+            activated at the later point(when the window is opened again) */
```

Comment:
I personally think that it is a "end of line" style comment fits better here then a block comment.